monitoring-as-code/src/dashboards/dashboard-standard-elements.libsonnet

```diff
@@ -35,10 +35,10 @@ local createRowTitles(sliKey, sliSpec) =
       sliType: sliSpec.sliType,
       title: sliSpec.title,
     },
-    rowTitleShort: '%(slo)s %(sliType)s (%(period)s)' % {
+    rowTitleShort: '%(slo)s: %(title)s' % {
```

Comment:
I think it would be better to refactor rowTitle and remove rowTitleShort as it's not used anywhere else.

Reply:
noted